app/models/fact_name.rb

```diff
@@ -23,7 +23,7 @@ class FactName < ApplicationRecord
 
   default_scope -> { order('fact_names.name') }
 
-  validates :name, :uniqueness => { :scope => :type }
+  validates :name, :length => { :maximum => 4096 }, :uniqueness => { :scope => :type }
```

Comment:
We have `validates_lengths_from_database` some lines above, so this change is not necessary. It'll pick up the (new) size automatically.

Reply:
OK. I've amended the commit to remove this redundant check.

(FWIW this is my first time ever doing anything related to Ruby/RoR so I was expecting issues like this.)